docs/source/start.ipynb

```diff
@@ -288,16 +290,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dt = dt.reset_semantic_tags()\n",
-    "dt"
+    "df.ww.reset_semantic_tags()\n",
+    "df.ww"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Set Index and Time Index\n",
-    "At any point, you can designate certain columns as the DataTable's `index` or `time_index` with the methods [set_index](generated/woodwork.datatable.DataTable.set_index.rst) and [set_time_index](generated/woodwork.datatable.DataTable.set_time_index.rst). These methods can be used to assign these columns for the first time or to change the column being used as the index or time index.\n",
+    "At any point, you can designate certain columns as the Woodwork `index` or `time_index` with the methods `set_index` and `set_time_index`. These methods can be used to assign these columns for the first time or to change the column being used as the index or time index.\n",
```

Comment:
Would it be useful to keep the hyperlinks to the methods?

Reply:
Yeah, we can keep those. I was thinking I couldn't do that yet since the API Ref doesn't exist, but I see now that I should be able to keep them.